pages/index.md

```diff
@@ -17,4 +17,6 @@ We will accomplish our mission by:
 * being design-centric, agile, open, and data-driven
 * deploying tools and services early and often
 
+We know, there is a lot of information on this site, which can be intimidating. Federal hiring processes are more complex than most other places, so we want to make sure you have all the information you need. We're happy you're interested in joining the team!
```

Comment:
Good call. I like the idea of more clearly communicating the pages' hierarchy of importance (i.e., which are "required" and which are simply there for reference). If there are pages that must be read before applying, though, I'd like us to ensure that site visitors encounter them en route to the application, as we can't guarantee that they'll land on this page first.

I share your assessment of those three pages (including the "maybe" about the "Who we are hiring" page.) @rooneywp, what do you think?